crates/node/core/src/args/engine.rs

```diff
@@ -35,7 +35,7 @@ pub struct EngineArgs {
     pub caching_and_prewarming_disabled: bool,
 
     /// Enable the parallel sparse trie in the engine.
-    #[arg(long = "engine.parallel-sparse-trie", default_value = "false")]
+    #[arg(long = "engine.parallel-sparse-trie", default_value = "true")]
```

Comment:
let's do in a way similar to https://github.com/paradigmxyz/reth/pull/15764, so that default in the code is that it's enabled, and if `parallel_sparse_trie_disabled == true`, then we use serial one.